### PR TITLE
Tzeng/130 inventory alphabetical sorting

### PR DESCRIPTION
**Pull request by aitzeng**
## Description

_Give a brief context and description of the code change. Highlight anything you want reviewers to pay attention to._
Added asc/desc to the Name button to organize display. Also updated state names to help readability. 

_The PR should have enough information so someone coming in fresh gets a sense of what's going on, but the details of the overall project can be documented on project.digitalaidseattle.org_

## What type of PR is this? (check all applicable)

- [] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

https://das-ph-inventory-tracker.atlassian.net/browse/PIT-130

- Related Issue #
- Closes #130

## QA Instructions, Screenshots, Recordings

N/A

### CODE REVIEW

File: src/pages/inventory/index.tsx [hunk 1]
```diff
@@ -154,15 +173,14 @@ const Inventory = () => {
       if (!response.ok) {
         throw new Error(response.statusText);
       }
-      console.log(response);
       const data = await response.json();
-      console.log(response);
+      console.log(data.value)
```
Comment (bospoort) [inline src/pages/inventory/index.tsx:177]: Nit: can we remove the console.logs?

File: src/pages/inventory/index.tsx [hunk 2]
```diff
@@ -172,6 +190,7 @@ const Inventory = () => {
 
   useEffect(() => {
     if (type || category || status || search) {
+      // console.log('database_connection_string:', process.env.DATABASE_CONNECTION_STRING);
       const handler = setTimeout(() => {
```
Comment (bospoort) [inline src/pages/inventory/index.tsx:194]: remove commented out console log

File: src/pages/inventory/index.tsx [hunk 3]
```diff
@@ -180,10 +199,15 @@ const Inventory = () => {
       };
       // handleFilter()
     } else {
+      // console.log('database_connection_string:', process.env.DATABASE_CONNECTION_STRING);
       fetchData();
```
Comment (bospoort) [inline src/pages/inventory/index.tsx:203]: Another

**Review by bospoort — CHANGES_REQUESTED** [review]
Looks good. A few remnants.

File: src/pages/inventory/index.tsx (hunk 1 above)
Comment (aitzeng) [reply src/pages/inventory/index.tsx:177]: Removed

File: src/pages/inventory/index.tsx (hunk 2 above)
Comment (aitzeng) [reply src/pages/inventory/index.tsx:194]: Removed

File: src/pages/inventory/index.tsx (hunk 3 above)
Comment (aitzeng) [reply src/pages/inventory/index.tsx:203]: Removed

**Review by pingchenchan — APPROVED** [review]

**Review by bospoort — APPROVED** [review]

**Pull request merged**